changelog.txt

```diff
@@ -1,3 +1,55 @@
+-------------------------------------------------------------------------------
+Version 0.67 - released xxx 2022
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* The backend code for MIDI playback has been changed and real pause logic has been
+  re-enabled. Please be aware that things such as application volume and audio quality
+  may change depending on your configuration from ≤0.66
+* The original demo gameplay video now plays when the CorsixTH main menu is idle (this
+  can be disabled in the configuration file)
+* File browser menus now have an OK button (at last!)
+* Selecting an existing file in the Save Game/Map browsers will now update the text
+  input box to that file name
+
+# Changes
+* Handymen now always stay in their assigned plots, except when going to rest
+* Staff members now have a maximum salary, which will be configurable in custom levels
+* The game will now always use the GP’s choice of next diagnosis room for the patient,
+  if available
+* Queueing patients are now shown when hovering a door
+* Epidemics can no longer fail because an infected patient left before the epidemic was
+  revealed
+* Balance changes to drug improvement research
+* Check for updates can now be disabled in Settings
+* “Pause” now shows up when the “Build while pause” setting is on and the game is paused.
+* The gamelog file now includes a timestamp
+* Many, many, refactors of the code to make it better to understand for future developers
+
+# Translations
+* Chinese Traditional and Simplified translations have been updated. Thanks @lokpro
+* Russian has been updated. Thanks Matt1
+* Fixed a bug where CJK languages could crash the game
+* Contact methods to reach the CorsixTH Team have been updated
+
+Bug Fixes
+* Fixed an instance where objects could disappear from the map
+* Fixed a bug where the empty action queue message was prevented from happening
+* Fixed a bug where dying patients became undead and would have a habit of ruining everything 
+* Fixed broken dynamic info during epidemics
+* Handymen will no longer try to repair machinery while a patient is using it
+* Psychiatrist couch has less chance of causing blocked paths
+* Roujin’s challenge is now properly applied (read: harder)
+* Serious Radiation is no longer contagious
+* Room blueprints can no longer go beyond external walls
+* Fixed incorrect button sounds in the Staff Management window
+* Fixed back button behaviour in Font Picker
+* Fixed a rare case where the next VIP date may be scheduled in the past
+* Fixed a rare case where awarded research points could be negative
+* Fixed a rare case of the balance sheet Hulking out
```

Comment:
There is a fallback for this line if we can't use the word Hulk

Reply:
cute but turning green is much clearer

Reply:
I agree, I had no idea what it meant until I read the comment by TheCycoONE